### ISSUE TITLE
Unable to find the information about cell status about XGboost

### ISSUE THREAD

**Issue by KarlKaise** [jyxi7676/SiftCell#4]
Hello,

I ran following code:SiftCellBoost("path/",threshold = 200)  #
Got this output:
[1] "Read DGE files"
[1] "Detect positive labeles"
Warning message in asMethod(object):
“sparse->dense coercion: allocating vector of size 3.2 GiB”
[1] "Detect Highly Variable Genes"
Warning message in asMethod(object):
“sparse->dense coercion: allocating vector of size 2.1 GiB”
[1] "Principlal Component Analysis"
[1] "Run XGBoost"
Warning message in system(command, intern = T):
“running command 'python /scicore/home/doetsch/kaiser0001/miniforge3/envs/R_Env_2025_4_4_2/lib/R/library/SiftCell/runXGboost.py /scicore/home/doetsch/kaiser0001/Sc_ChP_2024_full_processing/cell_ranger_sc_ChP/Cell_Ranger_output/Seurat_objects/Sift_empty_droplets_analysis/Output/John//labeledData.csv /scicore/home/doetsch/kaiser0001/Sc_ChP_2024_full_processing/cell_ranger_sc_ChP/Cell_Ranger_output/Seurat_objects/Sift_empty_droplets_analysis/Output/John//predData.csv /scicore/home/doetsch/kaiser0001/Sc_ChP_2024_full_processing/cell_ranger_sc_ChP/Cell_Ranger_output/Seurat_objects/Sift_empty_droplets_analysis/Output/John/ Sample' had status 1”
character(0)
attr(,"status")
[1] 1
[1] "Done!"

Than I ran this code as shown in the tutorial:

cell = read.table("path/CellContaining.txt",header = F)[,1]
df.cell.org = df.cell[df.cell$orig.ident=='SampleOrg',c("nCount_RNA","tSNE_1","tSNE_2","UMAP_1","UMAP_2")]
df.cell.org$SiftCellBoost = "Soup"
df.cell.org[match(paste0("SampleOrg_",cell),rownames(df.cell.org)),"SiftCellBoost"] = "Cell"
ggplot(df.cell.org,aes(tSNE_1,tSNE_2,color = SiftCellBoost))+geom_point(size=0.5,alpha=0.3)+theme_bw()+theme(legend.position = "bottom")

And got this error message:
Error in file(file, "rt"): cannot open the connection
Traceback:

1. file(file, "rt")
2. .handleSimpleError(function (cnd) 
 . {
 .     watcher$capture_plot_and_output()
 .     cnd <- sanitize_call(cnd)
 .     watcher$push(cnd)
 .     switch(on_error, continue = invokeRestart("eval_continue"), 
 .         stop = invokeRestart("eval_stop"), error = NULL)
 . }, "cannot open the connection", base::quote(file(file, "rt")))

The XGboost analysis didnt create any file with soup/nonsoup information just updated labeled.csv file with few training cells with the label.

I would be grateful for any information as how can I get this cell status for all cells in the dataset not only training set.

Thank you

**Comment by EYNOCATTFOS** [comment]
have u solved this problem?
